parser_library/src/processing/preprocessors/preprocessor_utils.cpp

```diff
@@ -107,15 +98,19 @@ std::vector<semantics::preproc_details::name_range> get_operands_list(
         if (operands.empty())
             break;
 
-        column_offset += trimmed;
-        auto [op, extracted_length] = extract_operand_and_argument(operands);
+        op_column_start += trimmed;
+
+        auto operand_view = extract_operand_and_argument(operands);
+        std::string operand;
+        operand.reserve(operand_view.length());
+        std::remove_copy_if(operand_view.begin(), operand_view.end(), std::back_inserter(operand), isspace);
```

Comment:
[](unsigned char c){ return isspace(c); }

technically isspace is not defined for values not representable by unsigned char, which is an issue on platform with signed char.

Reply:
and that I think about it, we should probably review other uses of isspace... because we really only want to find blank ' ' and not whitespace characters in general...